python/hsfs/feature_group.py

```diff
@@ -140,6 +140,7 @@ def __init__(
         self._notification_topic_name = notification_topic_name
         self._deprecated = deprecated
         self._feature_store_id = featurestore_id
+        self._feature_store = None
```

Comment:
Do we need this parameter? I think we are not using it.

Reply:
It is required, otherwise `fg.feature_store` will fail if setter is not called before.